gocse/main.go

```diff
@@ -65,7 +65,10 @@ func main() {
 	// the driver only uses this for encryption information,
 	// not to enforce schema constraints
 
-	schema := schema.CreateJSONSchema(dataKeyBase64)
+	schema, err := schema.CreateJSONSchema(dataKeyBase64)
```

Comment:
Question: is it common in golang to name variables the same as a package from which you're calling a method?

Reply:
This is the only use, so shadowing isn't a problem. However, good point. I've changed it.

Question: Do you actually mean to say "should we change the name so the package isn't shadowed"?

Reply:
I was wondering whether it was common practice in golang, not that it was a problem. If common practice, it would make perfect sense to all target personas, but otherwise could be confusing to a beginner.

Reply:
I can't honestly find a definitive answer, but I'd say the correct answer is to either alias the package name or change the variable name so there's no shadowing.

Personally, I'd go with changing the variable name rather than aliasing the package. I think this is least surprising and easier to read and keeps it clear where functionality is being imported from.

Reply:
SGTM